src/main/java/uk/gov/hmcts/reform/sendletter/tasks/MarkLettersPostedTask.java

```diff
@@ -51,9 +54,9 @@ public MarkLettersPostedTask(
     }
 
     @SchedulerLock(name = TASK_NAME)
-    @Scheduled(cron = "${tasks.mark-letters-posted}")
+    @Scheduled(cron = "${tasks.mark-letters-posted}", zone = EUROPE_LONDON)
     public void run() {
-        if (!ftpAvailabilityChecker.isFtpAvailable(LocalTime.now())) {
+        if (!ftpAvailabilityChecker.isFtpAvailable(LocalTime.now(ZoneId.of(EUROPE_LONDON)))) {
```

Comment:
Maybe this logic of using London time should live inside the `ftpAvailabilityChecker`?

Reply:
can be. was considering about this

Reply:
Quickly stabbed and inspected it is not as trivial as passing local time with zone id as it is done here. Each time used in checker then has to be converted to `OffsetTime` instead

Reply:
at least it could be tested